pkgs/applications/misc/wmenu/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromSourcehut
+, pkg-config
+, cmake
+, meson
+, ninja
+, cairo
+, pango
+, wayland
+, wayland-scanner
+, wayland-protocols
+, libxkbcommon
+, scdoc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wmenu";
+  version = "0.1.2";
+
+  src = fetchFromSourcehut {
+    owner = "~adnano";
+    repo = "wmenu";
+    rev = version;
+    hash = "sha256-mS4qgf2sjgswasZXsmnbIWlqVv+Murvx1/ob0G3xsws=";
+  };
+
+  nativeBuildInputs = [ pkg-config meson cmake ninja ];
+  buildInputs = [ cairo pango wayland wayland-protocols libxkbcommon scdoc ];
+
+  meta = with lib; {
```

Comment:
as per contribution guidelines

`meta.maintainers must be set.`

Reply:
Who should i set as maintainer? Myself?